app/views/app_configs/_form.erb

```diff
@@ -48,6 +48,7 @@
 
   <fieldset>
     <legend>Reservation Settings</legend><br />
+    <%= f.input :requests_affect_availability, label: 'Allow requests to block equipment availability?', hint: 'When enabled, reservation requests will block equipment item availability before the request is approved.' %>
```

Comment:
I think "affect" is good -- I only used block because it was the term used by the feature requester